handler/index.js

```diff
@@ -3,9 +3,12 @@ module.exports = {
     actions: {
         current_time: require('./action_current_time'),
         current_news: require('./action_current_news'),
+        subscribe: require('./payload_subscribe'),
     },
     payloads: {
         report_start: require('./payload_report_start'),
         fragment_next: require('./payload_fragment_next'),
+        subscribe: require('./payload_subscribe'),
+        unsubscribe: require('./payload_subscribe'),
```

Comment:
Please consider using distinctive function for showing the list and changing the subscriptions